### PR TITLE
Add helper function to get stream UUIDs from wrapped Skip service

### PR DESCRIPTION
**Pull request by bennostein**
See second commit for payoff in client code

### CODE REVIEW

File: skipruntime-ts/examples/utils.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@ export class SkipHttpAccessV1 {
 
   constructor(
     private streaming_port: number = 8080,
-    private control_port: number = 8081,
+    control_port: number = 8081,
```
Comment (beauby) [inline skipruntime-ts/examples/utils.ts:28]: Is this change needed?

**Review by beauby — APPROVED** [review]
LGTM

File: skipruntime-ts/examples/utils.ts (hunk 1 above)
Comment (bennostein) [reply skipruntime-ts/examples/utils.ts:28]: Yes, otherwise that's recognized as an unused field by our JS tooling.  Without `private`, it's just a local variable argument of the constructor.
